src/pages/ProductListing/ProductsList.jsx

```diff
@@ -1,7 +1,18 @@
-import React from 'react'
+import {products} from "../../backend/db/products"
```

Comment:
I think you need to fetch all the products from backend instead of just importing them from the DB folder.

Reply:
Thank you for pointing it out. I have fetched it now.